example/user2/entry.js

```diff
@@ -0,0 +1 @@
+console.log("I'm user2! I like inline sourcemaps.");
```

Comment:
So what does user1 like then? I bet user1 likes long walks accross a fizzbuzz main loop, or programming languages that are palindromes.

Reply:
Close! She actually likes long walks across fizzbuzz implemented with tail recursion.